resources/index.html

```diff
@@ -84,6 +84,7 @@
 		* [A Comparison of Modern Graphics APIs](https://alain.xyz/blog/comparison-of-modern-graphics-apis)
 		* [Scratchapixel](https://www.scratchapixel.com)
 		* [The Graphics Codex](https://graphicscodex.com/)
+		* [Raytracing in one week-end series](https://raytracing.github.io/)
```

Comment:
Hmm would it make sense to create a new heading for some `Raytracing` section?

Reply:
Oh I thought the sections were "fixed". I guess adding a "Raytracing" section makes sense^^